schemas/test/en/test_array_type_definitions.json

```diff
@@ -0,0 +1,80 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "Other input fields",
+    "theme": "default",
+    "description": "A questionnaire to test definitions (array type).",
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "questionnaire_flow": {
+        "type": "Linear",
+        "options": {
+            "summary": {
+                "collapsible": false
+            }
+        }
+    },
+    "sections": [
+        {
+            "id": "default-section",
+            "groups": [
+                {
+                    "id": "definition-group",
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "definition-block",
+                            "question": {
+                                "id": "question",
+                                "title": "Do you connect a LiFePO4 battery to your <em>photovoltaic system</em> to store surplus energy?",
+                                "type": "General",
+                                "definitions": [
+                                    {
+                                        "title": "What is a photovoltaic system?",
+                                        "contents": [
```

Comment:
Could we have two descriptions in the array, so it is visibly different from the `test_question_defintion`

Reply:
Two definitions? We are removing support for multiple definitions so don't really want to introduce multiple here

Reply:
Ohhh I see, did not know that. 👍